scripts/upgrade

```diff
@@ -202,11 +202,7 @@ systemctl daemon-reload
 #=================================================
 ynh_script_progression --message="Starting a systemd service..." --weight=1
 
-ynh_systemd_action --service_name=$app --action="start" --log_path="systemd" --timeout=15
-
-# Why do i have to do this again to pickup the new systemd config???
-systemctl daemon-reload
-systemctl restart jellyfin
+ynh_systemd_action --service_name=$app --action="restart" --log_path="systemd" --timeout=15
```

Comment:
Not sure that's enough. It wasn't picking up the new systemd override with just one call. I'll test tomorrow and report back.

Reply:
Its enough from initial testing